README.md

```diff
@@ -62,6 +62,19 @@ stable ![status](https://img.shields.io/badge/status-stable-brightgreen) )
 The status follows the
 [Kubernetes API versioning schema](https://kubernetes.io/docs/reference/using-api/#api-versioning).
 
+## Community
+
+Please find details on regular hosted community events in the [keptn/community repo](https://github.com/keptn/community) as well as our Slack channel(s) in the [CNCF Slack workspace.](https://cloud-native.slack.com/messages/keptn/)
+
+## Governance
+
+- [Community Membership](https://github.com/keptn/community/blob/main/community-membership.md)
+   It provides guidelines for community engagement, contribution expectations, and the process for becoming a community member at different levels. Maintained by Keptn maintainers.
```

Comment:
Please reduce the line length to max 120 characters, also max one sentence is allowed on a single line